File: python/copy_directory/function.py
def copy_directory(source_dir: str, destination_dir: str) -> bool:
    import os
    import shutil

    # Delete the destination directory if it already exists
    if os.path.exists(destination_dir):
        shutil.rmtree(destination_dir)

    # Copy the source directory to the destination directory
    shutil.copytree(source_dir, destination_dir)
    return True

File: javascript/copyDirectory/function.js
function copyDirectory(sourceDir, destinationDir) {
    const fs = require('fs');
    const path = require('path');

    // Delete the destination directory if it already exists
    if (fs.existsSync(destinationDir)) {
        fs.rmSync(destinationDir, { recursive: true, force: true });
    }

    // Copy the source directory to the destination directory
    fs.cpSync(sourceDir, destinationDir, { recursive: true });
    return true;
}
module.exports = { copyDirectory };
module.exports = { copyDirectory };